### ISSUE TITLE
Making file on output

### ISSUE THREAD

**Issue by Scorpovi4** [westmonroe/pdf-puppeteer#15]
How to save returned to callback to file with custom name? And how plugin upload images, does need set the path as "base" for it or not?

**Comment by joehiggs** [comment]
Changing the filename depends where you are downloading the file from. I'm adding an html example to the /example folder right now, should be up very soon. If you are downloading to a local file system, however, use the `path` option for puppeteer's `page.pdf` function. [More on that can be found here](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagepdfoptions). As a reminder, the args for the page.pdf function are passed through the `options` parameter of pdf-puppeteer's `convertHTMLToPDF` function.

For your other question, I'm not quite sure what you're asking... Could you rephrase?

**Comment by Scorpovi4** [comment]
How I try to convert and what I have on output.
```
                               const fsPath = require('fs-path');
                               const convertHTMLToPDF = require("pdf-puppeteer");			
                               const callback = function (pdf) {					
					fsPath.writeFile($.outputFolder + 'pdf/' + name + '.pdf', pdf, function(err){
						if(err) throw err;
					});
				}
				convertHTMLToPDF(html, callback, null, null, false).catch(err => {
					console.log(err);
				});
```

Output [http://prntscr.com/n5puwi](url)

The same result on regular fs method stream to file... This module [https://www.npmjs.com/package/html-pdf](url) more clear how to use it and convert but not so correctly because phantom.js doesn't support some css rules...
